crates/subspace-networking/Cargo.toml

```diff
@@ -35,7 +35,8 @@ features = [
     "dns-tokio",
     "gossipsub",
     "identify",
-    "identity-hashing-for-peer-id",
+    # TODO: disable this feature until the hardfork
```

Comment:
So we can revert override in workspace `Cargo.toml` as well? Also I don't think the comment is correct, I don't think we'll need a feature in the end at all.